alvr/client_openxr/src/extra_extensions/multimodal_input.rs

```diff
@@ -57,7 +57,7 @@ pub struct MultimodalMeta {
 
 impl MultimodalMeta {
     pub fn new<G>(
-        session: xr::Session<G>,
+        session: &xr::Session<G>,
```

Comment:
Well almost good, but going full nitpicky, i say you should not use a reference here since later in this function you have to call `to_owned()`. Same for `FaceTrackerPico::new()`. Maybe I should have been more clear.

Reply:
Oh, so as I understand, I should avoid `to_owned()` and clone value beforehand instead if I need to move it?

Reply:
if you need to move the value, use a move parameter. if you need only references, then you use a reference parameter. it's better to always expose to the caller any copies. This is for `MultimodalMeta::new()` and `FaceTrackerPico::new()`. I would make an exception maybe for functions made to abstract common code where I would use a reference regardless (like `get_instance_proc`, but actually this function doesn't need to own the Session so it doesn't matter)